src/server/app.ts

```diff
@@ -41,128 +41,131 @@ declare module "express" {
   }
 }
 
-let app = express();
-app.disable("x-powered-by");
+export default function createApp(serverSettings: ServerSettings, settingsManager: SettingsManager, version: string): Express {
```

Comment:
This file is just a refactor, from module scope variable to function that creates this variable with explicit dependencies